dom0/sd-dom0-files.sls

```diff
@@ -104,11 +97,22 @@ dom0-install-securedrop-workstation-dom0-config:
       - file: dom0-workstation-rpm-repo
 {% endif %}
 
-# Hide suspend/hibernate options in menus in prod systems
-{% if d.environment == "prod" or d.environment == "staging" %}
-dom0-disable-unsafe-power-management-xfce:
-  cmd.script:
-    - name: salt://update-xfce-settings
-    - args: disable-unsafe-power-management
-    - runas: {{ gui_user }}
-{% endif %}
+dom0-environment-directory:
+  file.directory:
+    - name: /var/lib/securedrop-workstation/
+    - mode: 755
+    - makedirs: true
+
+dom0-remove-old-environment-flag:
+  file.tidied:
+    - name: /var/lib/securedrop-workstation/
+    - require:
+      - file: dom0-environment-directory
+
+dom0-write-environment-flag:
+  file.managed:
+    - name: /var/lib/securedrop-workstation/{{ d.environment }}
+    - mode: 644
+    - replace: False
+    - require:
+      - file: dom0-remove-old-environment-flag
```

Comment:
This is a flag file approach that allows us to use `ConditionPathExists` in systemd units to disable or enable services depending if we're in dev, prod, etc.  It's consistent with what we're using for vms; I have filed an issue to suggest we use .preset files instead (#1052 ), but this is an "on the way" option for now.